cluster/pulumi/infra/grafana-dashboards/canton/sequencer_messages.json

```diff
@@ -248,6 +249,23 @@
           "range": true,
           "refId": "A",
           "useBackend": false
+        },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "prometheus"
+          },
+          "disableTextWrap": false,
+          "editorMode": "code",
+          "expr": "sum(rate(daml_sequencer_block_events_total{namespace=\"$namespace\", job=\"$job\"}[1h] offset 30d))",
```

Comment:
isn't doing anything over 30 days super slow in this dashboard? I'm a bit wary of making it even more expensive.

Reply:
sorry, my message was dumb. it's 30 days ago, not last 30 days. so it is the same interval but 30 days ago

Reply:
I executed it a few times and didn't notice any downgrade compared to the current setup.
We have lots of other queries that pull in lots of series, this will be just a single series basically.

> If round cycles don't overlap which they won't in general it's also not representative of anything.

this is the hourly rate (messages per second in the last hour), so rounds don't really matter